be-golang-plain/Jenkinsfile

```
@@ -0,0 +1,87 @@
+// build params
+def projectId = env.PROJECT_ID
+def componentId = env.COMPONENT_ID.toLowerCase()
+def packageName = env.PACKAGE_NAME
+def gitUrlHttp = env.GIT_URL_HTTP
+def gitUrlSsh = env.GIT_URL_SSH
+def group = "org.osi"
+
+def credentialsId = projectId + "-cd-cd-user-with-password"
```

Going forward, could you use Groovy interpolation? We use that already elsewhere so it would be nice for consistency:
```suggestion
"${projectId}-cd-cd-user-with-password"
```